actions/get_attachments.py

```diff
@@ -0,0 +1,12 @@
+from lib.actions import BaseAction
+
```

Comment:
This looks like the same code as what's above except no ID. Might be easier/cleaner to create a common function in the BaseAction and just call it from both action implementation files, that way there is less copy/paste going on?

Reply:
Great point - trouble when you develop them separately and then forget to work out what was common!